continuous-integration/windows/build.py

```diff
@@ -30,9 +21,8 @@ def _build(root):
 
 
 def _main():
-    options = _options()
-
-    _build(options.root)
+    root = Path(__file__).resolve().parents[2]
```

Comment:
I see we do this twice in two different scripts, could we somehow centralize this functionality?

Reply:
Not sure if it's worth the added complexity. It's a very simple one-liner, and we repeat this kind of thing all the time in the equivalent bash scripts.

Reply:
Ok, we can probably refactor it later if we need to